examples/DynamicPage/DynamicPage.ino

```diff
@@ -78,6 +78,7 @@ bool handleAcs(HTTPMethod method, String uri) {
     else {
       return false;    // Not found to accessing exception URI.
     }
+    currentUri = String("");
```

Comment:
This statement will never be reached since it is below an if else with a return in it. Besides, it defies the check. Why would this statement be beneficial?